.cirrus.yml

```diff
@@ -448,6 +448,7 @@ apiv2_test_task:
 compose_test_task:
     name: "compose test on $DISTRO_NV"
     alias: compose_test
+    only_if: &not_docs $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*'
```

Comment:
This should just be simply `only_if: *not_docs`

Reply:
ty!